lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -8197,11 +8181,7 @@ public String updateStudent() {
 			// Create or update a task
 			String reference = "/lessonbuilder/item/" + itemId;
 			String title = getPage(page.getPageId()).getTitle() + " - " + messageLocator.getMessage("simplepage.student-contents-task-title");
-			if (this.studentContentsCreateTask) {
-				createOrUpdateTask(reference, getCurrentSiteId(), title, null);
-			} else {
-				updateTask(reference, title, null);
-			}
+			createOrUpdateTask(reference, getCurrentSiteId(), title, null);
```

Comment:
Could we add a comment here?

// Dashboard task widget:  When a student content item is added create a task item for the site members.